packages/perseus-editor/src/widgets/radio/radio-option-settings.tsx

```diff
@@ -0,0 +1,133 @@
+import Button from "@khanacademy/wonder-blocks-button";
+import {TextArea} from "@khanacademy/wonder-blocks-form";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import Pill from "@khanacademy/wonder-blocks-pill";
+import {sizing, spacing} from "@khanacademy/wonder-blocks-tokens";
+import {HeadingXSmall} from "@khanacademy/wonder-blocks-typography";
+import trashIcon from "@phosphor-icons/core/bold/trash-bold.svg";
+import * as React from "react";
+
+import PerseusEditorAccordion from "../../components/perseus-editor-accordion";
+
+import styles from "./radio-option-settings.module.css";
+import {RadioStatusPill} from "./radio-status-pill";
+
+import type {PerseusRadioChoice} from "@khanacademy/perseus-core";
+
+type Props = {
```

Comment:
Can this also get a more descriptive name please? 
```suggestion
interface Props = {
```